app/lib/brexit_checker/account_jwt.rb

```diff
@@ -0,0 +1,30 @@
+class BrexitChecker::AccountJwt
+  def initialize(criteria_keys)
+    @criteria_keys = criteria_keys
+  end
+
+  def encode(key = ecdsa_key, algorithmn = "ES256")
+    JWT.encode payload, key, algorithmn
+  end
+
+private
+
+  attr_reader :criteria_keys
+
+  def payload
+    {
+      attributes: {
+        transition_checker_state: criteria_keys
+        key: key_uuid
+      }
+    }
```

Comment:
Sorry, I meant the payload should be:

```
{
  uid: "application's OAuth client ID",
  key: "application's key UUID",
  scopes: ["transition_checker"],
  attributes: { transition_checker_state: criteria_keys },
}
```

Reply:
amended [here](https://github.com/alphagov/finder-frontend/pull/2207/commits/8124e7dd2d372437a56ba367ad2eb65eb55d8cfa#diff-e4257d6c3bb3ff33ad32620a2d04b871R14-R21)